examples/src/main/java/alluxio/examples/S3ASpecificOperations.java

```diff
@@ -0,0 +1,172 @@
+
+  /**
+   * Test for creating multipart file.
+   */
+  public void createMultipartFileTest() throws IOException {
+    String testParent = PathUtils.concatPath(mTestDirectory, "createParent");
+    String testFile = PathUtils.concatPath(testParent, "createOpenLarge");
+    int numCopies;
+    try (OutputStream outputStream = mUfs.create(testFile, CreateOptions.defaults(mConfiguration)
+        .setCreateParent(true))) {
+      numCopies = 6 * Constants.MB / TEST_BYTES.length;
+      for (int i = 0; i < numCopies; ++i) {
+        outputStream.write(TEST_BYTES);
+      }
+    }
+    // Validate data written successfully and content is correct
+    try (InputStream inputStream = mUfs.openExistingFile(testFile)) {
+      byte[] buf = new byte[numCopies * TEST_BYTES.length];
+      int offset = 0;
+      while (offset < buf.length) {
+        int bytesRead = inputStream.read(buf, offset, buf.length - offset);
+        for (int i = 0; i < bytesRead; ++i) {
+          if (TEST_BYTES[(offset + i) % TEST_BYTES.length] != buf[offset + i]) {
+            throw new IOException("Content of the written file is incorrect");
+          }
+        }
+        offset += bytesRead;
+      }
+    }
+
+    // These two calls will trigger S3A list objects v1
+    if (!mUfs.isDirectory(testParent)) {
+      throw new IOException("The written directory does not exist or is file");
+    }
+    UfsStatus[] statuses = mUfs.listStatus(PathUtils.concatPath(mTestDirectory, "createParent"));
+    if (statuses.length != 1) {
+      throw new IOException("List status result is incorrect");
+    }
+  }
+
+  /**
+   * Test for creating and aborting multipart file.
+   */
+  public void createAndAbortMultipartFileTest() throws IOException {
+    String testFile = PathUtils.concatPath(mTestDirectory, "createAndAbort");
+    OutputStream outputStream = mUfs.create(testFile);
+    // Clean all intermediate multipart uploads
+    mUfs.cleanup();
+    if (getMultipartUploadList().size() != 0) {
+      throw new IOException("Failed to clean previous intermediate multipart uploads");
+    }
+
+    // Create a file but do not close it
+    int numCopies = 6 * Constants.MB / TEST_BYTES.length;
+    for (int i = 0; i < numCopies; ++i) {
+      outputStream.write(TEST_BYTES);
+    }
+    // Multipart upload that has in progress upload will not be aborted
+    ((S3ALowLevelOutputStream) outputStream).waitForAllPartsUpload();
+    int size = getMultipartUploadList().size();
+    if (size != 1) {
+      throw new IOException(String.format("Expected to have one intermediate multipart upload, "
+          + "but get %s in %s", size, mBucket));
+    }
+    mUfs.cleanup();
+    if (getMultipartUploadList().size() != 0) {
+      throw new IOException("Failed to clean the in progress multipart upload");
+    }
+  }
+
+  /**
+   * @return a list of intermediate multipart uploads
+   */
+  private List<MultipartUpload> getMultipartUploadList() {
+    return mUfs.getS3Client().listMultipartUploads(
```

Comment:
The common operations run well, but there are still `ClassNotFoundException: alluxio.underfs.s3a.S3AUnderFileSystem` because I need `S3AUnderFileSystem` to list the multipart upload and `((S3ALowLevelOutputStream) outputStream).waitForAllPartsUpload();`. Now working to address this issue.